.eslintrc.json

```diff
@@ -3,5 +3,9 @@
     "@integreat-app/eslint-config",
     "@integreat-app/eslint-config-react",
     "@integreat-app/eslint-config-jest"
-  ]
+  ],
+  "globals": {
+    "__DEV__": false,
+    "__VERSION__": false
```

Comment:
Should the version really be `false` here? In flow-typed/globals.js you define its type as string.